src/get-started/java-binding.md

```diff
@@ -2,8 +2,9 @@
 
 <!-- cspell:ignore testnet,prepend,JDWP -->
 
-**Exonum Java App** is an application that includes the Exonum framework
-and Java services runtime environment.
+**Exonum Java Binding** is a framework for building blockchain applications in Java, powered by Exonum.
```

Comment:
Despite the original part was about the Exonum Java _App_ to me it looks like it should be about Exonum Java _Binding_. Is this correct substitution?

Reply:
I reckon yes.